### PR TITLE
terravaloper1dmkdw7q23acc72knqv9yu46pz2m363705cnt27

### PR DESCRIPTION
**Pull request by Panvasi**
https://finder.terra.money/mainnet/validator/terravaloper1dmkdw7q23acc72knqv9yu46pz2m363705cnt27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +78,9 @@ _NOTE: Validators that have yet to put up a profile have been hidden. Get the fu
 | StakeBin                 | [Profile](./validators/terravaloper19ne0aqltndwxl0n32zyuglp2z8mm3nu0gxpfaw/README.md) | [Station Page](https://station.terra.money/validator/terravaloper19ne0aqltndwxl0n32zyuglp2z8mm3nu0gxpfaw) |
 | Terran One               | [Profile](./validators/terravaloper1krj7amhhagjnyg2tkkuh6l0550y733jnjnnlzy/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1krj7amhhagjnyg2tkkuh6l0550y733jnjnnlzy) |
 | Terran Stakers           | [Profile](./validators/terravaloper1uymwfafhq8fruvcjq8k67a29nqzrxnv9m6m427/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1uymwfafhq8fruvcjq8k67a29nqzrxnv9m6m427) |
-| TERRA-FIRMA              | [Profile](./validators/terravaloper1s6j9wxyprdsxmdf6mvsegsldvmtk0p92pa7aey/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1s6j9wxyprdsxmdf6mvsegsldvmtk0p92pa7aey) |
+| TERRA-FIRMA              | [Profile](./validators/terravaloper1qqu376azltyc5wnsje5qgwru5mtj2yqdhar97v/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1qqu376azltyc5wnsje5qgwru5mtj2yqdhar97v) |
```
Comment (emidev98) [inline README.md:81]: Hey, could you review this URL? since redirects to the main page of station

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ _NOTE: Validators that have yet to put up a profile have been hidden. Get the fu
 | Forbole                  | [Profile](./validators/terravaloper1jkqr2vfg4krfd4zwmsf7elfj07cjuzss30ux8g/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1jkqr2vfg4krfd4zwmsf7elfj07cjuzss30ux8g) |
 | FRESHLUNA                | [Profile](./validators/terravaloper1audgfvmgt0js54p3s8kj3r40uwej6vy2tv6rrw/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1audgfvmgt0js54p3s8kj3r40uwej6vy2tv6rrw) |
 | G'Lab                    | [Profile](./validators/terravaloper122jtp99q03vjdq2d63fgtmsjyndhkh3whaqaas/README.md) | [Station Page](https://station.terra.money/validator/terravaloper122jtp99q03vjdq2d63fgtmsjyndhkh3whaqaas) |
-| Galactic Punks Validator | [Profile](./validators/terravaloper15lsftv92eyssjwkh2393s0nhjc07kryqen2fqf/README.md) | [Station Page](https://station.terra.money/validator/terravaloper15lsftv92eyssjwkh2393s0nhjc07kryqen2fqf) |
+| Galactic Punks Validator | [Profile](./validators/terravaloper1e6w5qgzs5rzrz8ark25lagm2ga2h9n2tvgzpsl/README.md) | [Station Page](https://station.terra.money/validator/terravaloper1e6w5qgzs5rzrz8ark25lagm2ga2h9n2tvgzpsl) |
```
Comment (emidev98) [inline README.md:38]: Hey, could you review this URL? since redirects to the main page of station

**Review by emidev98 — CHANGES_REQUESTED** [review]

**Comment by Panvasi** [comment]
finder link: https://finder.terra.money/mainnet/validator/terravaloper1dmkdw7q23acc72knqv9yu46pz2m363705cnt27

station link: https://station.terra.money/validator/terravaloper1dmkdw7q23acc72knqv9yu46pz2m363705cnt27

**Comment by emidev98** [comment]
Cool! My bad I was pointing to the classic network.

**Pull request merged**